<%# 用来显示创建和修改活动经过服务端验证后得到的错误消息 %>
<head>
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/meyer-reset/2.0/reset.min.css">
</head>

<% provide(:title, "错误提示")%>
<%= render "layouts/activities/header"%>
<br/>
<br/>
<div>
<p class="loading-text">
    出错了 :(
</p>
<br/>
</div>

<div class="container">
<div class="row">
<div class="col-md-8 activity-show-card activity-show-card-1">
<% if @activity.errors.any? %>
<div class="activity-errors-warn" style="margin-top:10px;padding-top:5px;font-weight:600">
    尝试失败，可能的原因如下：<br/>
    <ul class="list-group">
    <% @activity.errors.full_messages.each do |msg|%>
        <li class="list-group-item list-group-item-danger" style="font-size:14px;">
        <%= msg%>
        </li>
    <%end%>
    </ul>
</div>
<% end%>
</div>
</div>
</div>

<br/>
<br/>
<br/>
<p style="text-align:center;">
<svg class="spinner" width="30px" height="30px" margin="9px" viewBox="0 0 66 66" xmlns="http://www.w3.org/2000/svg">
   <circle class="path" fill="none" stroke-width="6" stroke-linecap="round" cx="33" cy="33" r="30"></circle>
</svg>
</p>
<%= render "layouts/activities/self_footer"%>
